### ISSUE TITLE
Twitter X icon missing

### ISSUE THREAD

**Issue by t-chatoyan** [robcresswell/vue-material-design-icons#300]
Hi! Thanks for making this.

It would be very beneficial to add new Twitter X icon

**Comment by robcresswell** [comment]
Yeah this is definitely overdue an update. I'll get to it later today

**Comment by robcresswell** [comment]
I actually can't do anything here, the library I build from stopped receiving updates it seems: https://www.npmjs.com/package/@mdi/js
